binr/rahash2/rahash2.c

```diff
@@ -309,13 +307,13 @@ static int do_help(int line) {
 
 static void algolist() {
 	ut64 bits;
-	int i;
+	ut64 i;
 	eprintf ("Available Hashes: \n");
 	for (i = 0; i < R_HASH_NBITS; i++) {
 		bits = 1ULL << i;
 		const char *name = r_hash_name (bits);
 		if (name && *name) {
-			printf ("  %s\n", name);
+			eprintf ("  %s\n", name);
```

Comment:
Use printf here otherwise ppl will not be able to pipe or grep yhat thing without fd redirections which are always anoying